CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 2021-07-26 v0.6.1
```

Comment:
Are we doing a patch release in this PR? @jm-clius 

Usually we add it to the file and then once we cut release we add the actual release version and name. This process is decoupled from any individual PR.

Reply:
This should just be under the heading `## Next version`. No need to provide release summary or summary of supported `libp2p protocols`. We just add our `Changes` as we go along and compile them to release notes once we determine a release is necessary.

Reply:
resolves in 5a7869620e2a3dbd9d63e441b7cc21f29bb8c1ee.